topic/src/main/java/tech/ydb/topic/description/ConsumerPartitionInfo.java

```diff
@@ -0,0 +1,181 @@
+package tech.ydb.topic.description;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+import tech.ydb.core.utils.ProtobufUtils;
+import tech.ydb.proto.topic.YdbTopic;
+
+/**
+ * @author Nikolay Perfilov
```

Comment:
Who, me?

Reply:
For the most part yes) This class was copied from PartitionInfo. The protobuf specifications of them are almost same